web-components/src/components/inputs/new/ConfirmationCode/ConfirmationCode.tsx

```diff
@@ -0,0 +1,179 @@
+import React, {
+  forwardRef,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+} from 'react';
+
+import { cn } from '../../../../utils/styles/cn';
+import {
+  ConfirmationCodeProps,
+  ConfirmationCodeRef,
+  InputProps,
+} from './ConfirmationCode.types';
+
+/**
+ * A component that displays a confirmation code input field.
+ */
+export const ConfirmationCode = forwardRef<
```

Comment:
can we add a story for this new input component? It makes sense to always add stories for new components and this will be less work to do as part of https://github.com/regen-network/rnd-dev-team/issues/1210